### ISSUE TITLE
[Feature Request] Support for avatar option in create/edit project

### ISSUE THREAD

**Issue by MarcinNowak-codes** [terraform-providers/terraform-provider-gitlab#303]
In the GitLab's user interface it is possible to upload image as project's avatar. Nice to have the same feature with Terraform.

 In the project xanzy/go-gitlab is already issue [Support for avatar option in create/edit project](https://github.com/xanzy/go-gitlab/issues/229). This issue is created to cross track progress on it.

**Comment by armsnyder** [comment]
The xanzy/go-gitlab package supports the `avatar_url` field. Could that be used in the meantime?

**Comment by armsnyder** [comment]
UploadAvatar is in the latest go-gitlab version now.

**Comment by morsik** [comment]
Is there any update on this? Could be very nice to have this feature :)

**Comment by timofurrer** [comment]
It's currently blocked by https://gitlab.com/gitlab-org/gitlab/-/merge_requests/92604

**Comment by coutantal** [comment]
+1 for this feature. It seems that [92604](https://gitlab.com/gitlab-org/gitlab/-/merge_requests/92604) has been merged !
